### DISCUSSION TITLE
Does this still work? Can't get license

### DISCUSSION THREAD

**Discussion by zeh** [airsdk/Adobe-Runtime-Support#1813] [General]
I'm trying to get a license so I can get a adt.lic to update an old (free!) Android game of mine to 64bit. I tried going through the license on https://airsdk.harman.com/pricing and while I did get an email confirmation of my request, I never got the "invoice" they said they'd send me.

Is Harman still selling licenses? Or is the business dead?

And is the SDK working fine? It seems to work from my tests, but I didn't do any exhaustive QA, and the lack of responses sorta worries me.

**Comment by ajwfrost** [comment]
Yes it's still going! There are sometimes manual steps for the team when creating invoices, primarily if you're based in the US due to the tax regulations (i.e. whether we have to charge tax on the software depends on both where you are and on our own tax filing info) - this is in the process of being automated but it's currently still needing a manual step, so it can take a bit of time to sort out..

I'll chase up though to check that the team got the notification from the system for this...

"Fine" is probably a good description for the state of the SDK :-) just preparing our next release and there are other improvements we will be making, some bug fixes as well as usability enhancements and then some new features coming soon too.

thanks

**Reply by ajwfrost** [reply]
Hi again - I've just confirmed that we did receive the order so it's being processed, and we had sent out an automated email to you to confirm this ... actually I've just noticed that you did say above that you received the initial confirmation.

So this said "We will send out the invoice in due course" and sadly that doesn't mean it's within the next hour or so. It's usually within 24-48 hours... 

thanks

**Reply by zeh** [reply]
Thanks @ajwfrost. I suppose I assumed the invoice was automated as well, so after a day had passed, was a bit curious and wanted to double-check.

I'll wait for it, no rush.
